src/test/java/com/google/javascript/gents/singleTests/comments.js

```diff
@@ -40,7 +40,7 @@ var foo = function(deleted, notdeleted) { return deleted + notdeleted; };
  * @package {number}
  * @const {number}
  */
-var x;
+var x = 1;
```

Comment:
It will need to be split into more variables because most of these annotations are not compatible with each other. It defeats the purpose of this test to have many invalid annotations as the comment says `// The following comment should be mostly deleted`.
I wonder if we should just remove this test case since this is unlikely to compile.